libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -832,8 +841,8 @@ bool AP_AHRS::airspeed_sensor_enabled(void) const
 bool AP_AHRS::_airspeed_estimate(float &airspeed_ret, AirspeedEstimateType &airspeed_estimate_type) const
 {
 #if AP_AIRSPEED_ENABLED && AP_GPS_ENABLED
-    if (airspeed_sensor_enabled()) {
```

Comment:
This is the issues were fixing, this line checks the **PRIMARY** sensor is set to use and healthy.